### PR TITLE
Extend support for Snowflake

### PR DESCRIPTION
**Pull request by georgewoodhead**
## Description & motivation
Extend accelerator support for Snowflake. This includes:
- Creating new Jupyter notebooks to train and deploy the propensity to convert machine learning models with Snowpark for python
- Add instructions to connect Snowflake with Hightouch
- Make content more generic to work with both Snowflake and Databricks

Other improvements:
- Mention Fractional Attribution DPA in the next steps
- Add mention of how to benchmark the modelling

## Checklist
- [x] I have updated the README.md (if applicable)

### CODE REVIEW

**Review by rlh1994 — CHANGES_REQUESTED** [review]
Couple of things as I was working through, but otherwise looks good and clean!

- Can we make clear which schema their sessions should be connecting to (i.e. the one that has what tables, is it my main schema or my derived one?)
- It might be worth adding in as to why we would want to create this function to train our model given we just did it (i.e. because we should regularly re-train our model to avoid model drag, and this makes it easier to do so).
- Maybe in the configuration in later notebooks, unless there is any difference, just call out they only need to do this if they have not done it before/their setup has chagned from the previous steps?
- Can we add the final image to the introduction under what you will build as well as the dashboard image? I think it's great to show the things that you'll have by the end outside just the dashboard given how much of it is the model.
- As discussed on slack, if we can split the steps for databricks/snowflake under different sub-headers on the nav menu that would be ideal, one for @Jack-Keene ?

Other than that looks good!

**Comment by Jack-Keene** [comment]
By next level of header do you mean something like:

1. Predictive ML Modeling
   - Databricks
         - Model Training
         - Using Your Model
   - Snowflake
         - Model Training
         - Using Your Model

**Comment by rlh1994** [comment]
@Jack-Keene Yep

**Comment by Jack-Keene** [comment]
This should be pretty straightforward to do, I think all you need to do is add a 'databricks' and 'snowflake' folder to add subheaders. Happy to jump on a call if that doesnt work

File: config.toml [hunk 1]
```diff
@@ -2,5 +2,5 @@ title = "Composable CDP with Predictive ML Modeling"
 baseURL = "https://docs.snowplow.io/accelerators/composable-cdp-with-predictive-ml-modeling/"
 
 [params]
-  description = "Tutorial building a composable CDP with predictive ML modeling using Snowplow, Databricks and Hightouch"
+  description = "Tutorial building a composable CDP with predictive ML modeling using Snowplow, Databricks or Snowflake, and Hightouch"
   author = "Snowplow, George Woodhead"
```
Comment (Jack-Keene) [inline config.toml:6]: Can we add a parameter to disable intercom here: 
`intercom=false`

**Pull request merged**